src/MOI_wrapper.jl

```diff
@@ -0,0 +1,355 @@
+using LinearAlgebra
+import MathOptInterface as MOI
+using ManifoldsBase
+using ManifoldDiff
+
+include("qp_block_data.jl")
+
+const _FUNCTIONS = Union{
+    MOI.VariableIndex,MOI.ScalarAffineFunction{Float64},MOI.ScalarQuadraticFunction{Float64}
+}
+
+"""
+    struct VectorizedManifold{M} <: MOI.AbstractVectorSet
+        manifold::M
+    end
+
+Representation of points of `manifold` as a vector of `R^n` where `n` is
+`MOI.dimension(VectorizedManifold(manifold))`.
+"""
+struct VectorizedManifold{M} <: MOI.AbstractVectorSet
+    manifold::M
```

Comment:
Do we maybe want to restrict `M` to ba `<: AbstractManifold`?

Reply:
Done